src/test/kotlin/uk/gov/dluhc/printapi/messaging/SendApplicationToPrintMessageListenerIntegrationTest.kt

```diff
@@ -167,12 +155,8 @@ internal class SendApplicationToPrintMessageListenerIntegrationTest : Integratio
         sqsMessagingTemplate.convertAndSend(sendApplicationToPrintQueueName, payload)
 
         // Then
-        await.during(5, SECONDS).until {
-            val response = dynamoDbClient.scan(
-                ScanRequest.builder().tableName(dynamoDbConfiguration.printDetailsTableName).build()
-            )
-            assertThat(response.items()).isEmpty()
-            true
+        await.atMost(5, SECONDS).untilAsserted {
```

Comment:
Found that checking the repo would cause test always to pass even if the validation was skipped as the test doesn't set up all the other dependencies for it to succeed.  Also there is a race condition as the check is same as the initial state.

Found we can positively test for an error message that is directly related to the validation failure.